build.gradle

```diff
@@ -71,21 +89,30 @@ task testJar(type: Jar) {
   from sourceSets.test.output
 }
 
-task integrationTest(type: Test) {
-    description = 'Runs the integration tests.'
+task integrationTestCassandra(type: Test) {
+    description = 'Runs the integration tests for Cassandra.'
     group = 'verification'
-    testClassesDirs = sourceSets.integrationTest.output.classesDirs
-    classpath = sourceSets.integrationTest.runtimeClasspath
+    testClassesDirs = sourceSets.integrationTestCassandra.output.classesDirs
+    classpath = sourceSets.integrationTestCassandra.runtimeClasspath
     outputs.upToDateWhen { false }  // ensures integration tests are run every time when called
     shouldRunAfter test
+}
 
+task integrationTestJdbc(type: Test) {
+    description = 'Runs the integration tests for a JDBC database.'
+    group = 'verification'
+    testClassesDirs = sourceSets.integrationTestJdbc.output.classesDirs
+    classpath = sourceSets.integrationTestJdbc.runtimeClasspath
+    outputs.upToDateWhen { false }  // ensures integration tests are run every time when called
+    shouldRunAfter test
```

Comment:
Just wondering if this is needed. (since it is only executed on-demand basis, it's not needed?)